nrich-registry/src/main/java/net/croz/nrich/registry/data/customizer/RegistryDataFormConfigurationMappingCustomizer.java

```diff
@@ -42,21 +44,34 @@ private void registerRegistryFormConfiguration(Map<String, Class<?>> formConfigu
             String registryCreateFormId = String.format(RegistryDataConstants.REGISTRY_FORM_ID_FORMAT, registryClassName, RegistryDataConstants.REGISTRY_FORM_ID_CREATE_SUFFIX);
             String registryUpdateFormId = String.format(RegistryDataConstants.REGISTRY_FORM_ID_FORMAT, registryClassName, RegistryDataConstants.REGISTRY_FORM_ID_UPDATE_SUFFIX);
 
-            if (formConfigurationMap.get(registryCreateFormId) == null) {
-                Class<?> createClass = resolveClass(registryClassName, RegistryDataConstants.CREATE_REQUEST_SUFFIX);
+            formConfigurationMap.computeIfAbsent(registryCreateFormId, key -> resolveClassByPackage(registryClassName, RegistryDataConstants.CREATE_REQUEST_CLASS_NAME_FORMAT));
+            formConfigurationMap.computeIfAbsent(registryUpdateFormId, key -> resolveClassByPackage(registryClassName, RegistryDataConstants.UPDATE_REQUEST_CLASS_NAME_FORMAT));
+        });
+    }
 
-                formConfigurationMap.put(registryCreateFormId, createClass);
-            }
-            if (formConfigurationMap.get(registryUpdateFormId) == null) {
-                Class<?> updateClass = resolveClass(registryClassName, RegistryDataConstants.UPDATE_REQUEST_SUFFIX);
+    private Class<?> resolveClassByPackage(String registryFullClassName, String registrySuffixFormat) {
```

Comment:
Maybe rename `registrySuffixFormat` to `classNameFormat` or `requestClassNameFormat`. If we agree to rename it should be done for `resolveClass` method parameter also.